src/main/java/com/kodilla/ecommercee/OrderController.java

```diff
@@ -1,4 +1,47 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@CrossOrigin(origins = "*")
```

Comment:
Jeśli chodzi o @CrossOrigin(origins = "*") to mam co do tego mieszane odczucia.
Może to być dla testów natomiast nie powinno raczej iść na produkcję. Bardzo często repo git jest konfigurowane w taki sposób, że to co jest merge'owane do gałęzi master jest gotowe to wdrożenia na produkcję. W związku z tym bardzo łatwo potem jest zapomnieć, że zostawiliśmy taką gwiazdkę. 

@CrossOrigin(origins = {"http://localhost:8080", "http://localhost:88888", "http://localhost/", "http://domana-testowa.system.pl", "http://domena-produkcyjna.system.pl"} )

Teoretycznie można zostawić gwiazdkę o ile potem się wdraża jakieś ReverseProxy gdzie się nadpisuje te nagłówki i ustala inne, bezpieczniejsze od gwiazdki. Natomiast to już zależy od etapu wdrożenia konkretnego systemu. 

Tutaj możesz poczytać więcej jaką potencjalną dziurę bezpieczeństwa może to spowodować: 
https://stackoverflow.com/a/12014554